eagle-stores/src/pages/Profile/Profile.css

```diff
@@ -0,0 +1,57 @@
+:root {
+    --bg-color: #ededed;
+    --color: #113946;
+    --off-grey: #bcc6cc;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.profile {
+    width: 17.87rem;
+    background-color: var(--bg-color);
+    height: 21rem;
```

Comment:
Can give max or min width and height for responsiveness, can refactor later on

Reply:
So here I am using rem units so it's already responsive